### ISSUE TITLE
[🕹️] Build Pricing Calculator 

### ISSUE THREAD

**Issue by shnai0** [mfts/papermark#627]
### What side quest or challenge are you solving?

Many Papermark users, sharing whole data room using one link. The data room market is complicated and pricing structures also. 

To attract more website visitors and make they life easier we would to have this price calculator on our landing page papermark.io

Here is an example: https://firmroom.com/calculator

Points: 750

Provide proof that you've completed the task

You'll get access to the repo of papermark to open your PR 🤓

**Comment by syedbarimanjan** [comment]
I would like to work on this and want to know if you have a design for it.

/assign

**Comment by syedbarimanjan** [comment]
@shnai0 is there a design in your mind for this.

**Comment by shnai0** [comment]
@syedbarimanjan just added more extensive explanation, let me know if you need more help 😀

**Comment by 01vishu** [comment]
@syedbarimanjan is there any updates on this issue?

**Comment by adityadeshlahre** [comment]
Hey @syedbarimanjan please share updates and if you are not working please let us know I would like to work on it!
Thank You :slightly_smiling_face:

**Comment by Khaan25** [comment]
Hey @syedbarimanjan, any updates on this? I would like to work on it?

**Comment by Khaan25** [comment]
@chhavikant-01 Could you please remove your assignment? I've asked for it

**Comment by adityadeshlahre** [comment]
> /assign

@chhavikant-01 bruh please remove your assignment!
I asked @syedbarimanjan personally {he can confirm it} for this issue!
Thank You :slightly_smiling_face:

**Comment by syedbarimanjan** [comment]
> > /assign
> 
> @chhavikant-01 bruh please remove your assignment! I asked @syedbarimanjan personally {he can confirm it} for this issue! Thank You 🙂

Yeah I am leaving the issue to @adityadeshlahre. @chhavikant-01 please cooperate.

**Comment by Hitarthsheth07** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by DeepaPrasanna** [comment]
hey @Hitarthsheth07 , you cant do this without assigning yourself first

**Comment by Hitarthsheth07** [comment]
> hey @Hitarthsheth07 , you cant do this without assigning yourself first

Well, I just did!

**Comment by DeepaPrasanna** [comment]
> > hey @Hitarthsheth07 , you cant do this without assigning yourself first
> 
> Well, I just did!

I meant that is against the rule, you can see the issue is already assigned to somebody.
